src/zoid/buttons/util.js

```diff
@@ -18,6 +18,30 @@ export function determineFlow(props : ButtonProps) : $Values<typeof BUTTON_FLOW>
     }
 }
 
+export function mightUseQRPay(funding : $Values<typeof FUNDING>) : boolean {
+    if (funding === FUNDING.VENMO && !isIos() && !isAndroid()) {
+        return true;
+    }
+
+    return false;
+}
+
+export function showButtonLoading (targetElement : HTMLElement) : void {
+    const buttonElement = (targetElement.getAttribute('role') === 'button') ?
+        targetElement :
+        targetElement.closest('[role="button"]');
```

Comment:
We cannot use `closest()` since it doesn't work in all the browsers we support (IE11 😢 ) https://caniuse.com/?search=closest.

Reply:
doesn't webpack fix that? We use [arrow functions](https://caniuse.com/arrow-functions) everywhere?

Reply:
Things like arrow functions and const/let are easy to transpile and are safe to use. DOM apis like closest() would need to be polyfilled and that is not done automatically. We should figure out another selector to use instead of trying to use `closest()`